pkg/apis/postgres-operator.crunchydata.com/v1beta1/shared_types.go

```diff
@@ -256,3 +256,35 @@ func (meta *Metadata) GetAnnotationsOrNil() map[string]string {
 	}
 	return meta.Annotations
 }
+
+type AdditionalVolume struct {
```

Comment:
👍🏻 to this move. Watch out for conflict with https://github.com/CrunchyData/postgres-operator/commit/6b01868fd0fe32c8c6be99b7393b625ce5992312.